docs/src/http.md

```diff
@@ -95,11 +95,20 @@ to replace previously sent, unreceived subscription updates. See
 
 **Call:**
 
+```
+https://updates.push.services.mozilla.com/wpush/v1/...
+```
+
+
+
 If the client is using webpush style data delivery, then the body in
 its entirety will be regarded as the data payload for the message per
 [the WebPush
 spec](https://tools.ietf.org/html/draft-thomson-webpush-http2-02#section-5).
 
+> _**Note**_
+> Mozilla reserves the right to change the endpoint at any time. Please do not "optimize" by only storing the last token element of the URI. There will be tears.
```

Comment:
🤣 we should write "There will be tears" in our docs more often. Sums up a lot of software engineering well.